app.js

```diff
@@ -2,6 +2,11 @@ const recipeDiv = document.querySelector("#recipesDiv");
 checkedRadio = document.querySelector("input[name='ingredient']:checked");
 searchIngredient(checkedRadio);
 
+function getRandomInt(max) {
+  return Math.floor(Math.random() * Math.floor(max));
+}
+
+
```

Comment:
I think, I would prefer to make sure that there is a minimum number of cakes, that not only one is going to be displayed... return Math.random() * (max - min) + min;

Reply:
> I think, I would prefer to make sure that there is a minimum number of cakes, that not only one is going to be displayed... return Math.random() * (max - min) + min;
This method retuns a random integer from 0 to max-1. I use this number as a `from` parameter for API-call to randomise the cakes we get each time. When 0 is returned, it's not a problem then. Because we get the same cakes then, as it was before. 
The number of cakes displayed is always 10 for now, but I'll try to implement the caching-functionality and the "show more" button as well. 
This was just a quick-fix to have new cakes each time you change the radio-button choice.